tests/pytest/common.py

```diff
@@ -2,7 +2,6 @@
 from contextlib import contextmanager
 from functools import wraps
 from includes import *
-from packaging import version
```

Comment:
if it doesn't fix the issue with focal please revet

Reply:
This import is unused. It is dead code.